0023-backend-v3.md

```diff
@@ -0,0 +1,269 @@
+
+#### 2. The `.run` method (will change in BackendV3)
+
+The abstract interface definition of `BackendV2` requires that subclasses implement a `.run` method for execution, the following table shows some example of `BackendV2` implementations where the `.run` method plays an important role:
+
+|   Backendv2                            | Target                                                      | .run                                          |
+|--------------------------------------|-------------------------------------------------------------|-----------------------------------------------|
+|   `GenericBackendV2` (qiskit)          |   Customizable: built in backend init following user inputs | Calls `BasicSimulator.run` or `AerSimulator.run`  |
+|   `FakeBackendV2` (qiskit-ibm-runtime) |   Built from historical IBM hardware information (json)     | Calls `BasicSimulator.run` or `AerSimulator.run`  |
+|      `IBMBackend` (qiskit-ibm-runtime) |   Built from current IBM hardware information (json)        | Disabled (used to access real HW)             |
+| `AerSimulator` (qiskit-aer)            |   Defined by user, built from noise model or default        | Proprietary implementation of local simulator |
+| `AQTResource` (qiskit-aqt-provider)    |   AQT target                                                | 3rd-party AQT ion-trap system                 |
+                             
+`.run` used to be the public API for accessing IBM Quantum hardware, but this is no longer the case. Public access through `.run` has been revoked, and current backends in `qiskit-ibm-runtime` only support access through the `Sampler.run` and `Estimator.run` methods.
+
+Instead of a required `backend.run()`, the proposal for `BackendV3` implementations is to offer **primitive factory methods**: `backend.sampler()` and `backend.estimator()`. 
+
+#### 3. Extra functionality (small changes in BackendV3)
+
+As mentioned initially, this category includes both optional and custom functionality that is currently present in `BackendV2` implementations. The difference between optional and custom is that Qiskit offers tools to reason about optional functionality, while custom functionality requires custom tools to handle it (similar philosophy to the `Target` class).
+
+**Optional Functionality** -> `QubitProperties`
+
+For historical reasons, `QubitProperties` remains an optional property on both the `Target` and `BackendV2` objects. This adds redundancy and ambiguity, as any functionality that uses `QubitProperties` must query both and establish rules on what takes precedence over what. This can be easily fixed in `BackendV3` by no longer storing `QubitProperties` in the backend, and keeping them exclusively as an optional `Target` property. Querying these properties would still be possible in `BackendV3` for backwards compatibility by aliasing the target query.
```

Comment:
Aliasing them would solve the problem of them being out of sync, but it doesn't seem to address the ambiguity problem, which I view as the real problem, as I suspect that the average user will not understand that it's an alias, and still wonder why the heck the same information seems to be in two places.

I would favor making a breaking change here.